src/vs/workbench/parts/files/browser/fileActions.contribution.ts

```diff
@@ -169,6 +169,7 @@ registry.registerWorkbenchAction(new SyncActionDescriptor(RevertFileAction, Reve
 registry.registerWorkbenchAction(new SyncActionDescriptor(GlobalNewFolderAction, GlobalNewFolderAction.ID, GlobalNewFolderAction.LABEL), category);
 registry.registerWorkbenchAction(new SyncActionDescriptor(GlobalCompareResourcesAction, GlobalCompareResourcesAction.ID, GlobalCompareResourcesAction.LABEL), category);
 registry.registerWorkbenchAction(new SyncActionDescriptor(CloseFileAction, CloseFileAction.ID, CloseFileAction.LABEL, { primary: KeyMod.chord(KeyMod.CtrlCmd | KeyCode.KEY_K, KeyCode.KEY_W) }), category);
+registry.registerWorkbenchAction(new SyncActionDescriptor(OpenRecentlyClosedResourcesAction, OpenRecentlyClosedResourcesAction.ID, OpenRecentlyClosedResourcesAction.LABEL), category);
```

Comment:
Suggest to name this "Reopen Closed File" (as ST) and add it to the main menu under File > Open Recent as first entry with a separator to the rest.